website/docs/r/elb_loadbalancer.html.markdown

```diff
@@ -45,9 +45,9 @@ The following arguments are supported:
     Internal or External. Changing this creates a new elb loadbalancer.
 
 * `admin_state_up` - (Required) Specifies the status of the load balancer.
-    Value range: 0 or false: indicates that the load balancer is stopped. Only
-    tenants are allowed to enter these two values. 1 or true: indicates that
-    the load balancer is running properly. 2 or false: indicates that the load
+    Value range: false: indicates that the load balancer is stopped. Only
+    tenants are allowed to enter these two values. true: indicates that
+    the load balancer is running properly. false: indicates that the load
     balancer is frozen. Only tenants are allowed to enter these two values.
```

Comment:
Can we refine this with "Value range: false: indicates that the load balancer is stopped or frozen; true: indicates that the load balancer is running properly. Only tenants are allowed to enter these two values."